public class Main {
    public static void main(String[] args) {
        int a[]={30,20,10,40,0};
        try{
            int c=a[0]/a[4]; //if here any exception present then it will go to catch and execute
            // other statements doesnot executed
            System.out.println("division is "+c);
        try {
            System.out.println(a[8]); //if there is no exception before thi stmt then only it
            // handled otherwise it will not executed
        }
        catch (ArrayIndexOutOfBoundsException e){
            System.out.println("Index is invalid");
        }

        }
        catch (ArithmeticException e){
            System.out.println("denominator should not be 0");
        }

        System.out.println("Bye");
    }
}
